scripts/context_retrieval/wrangle_llama_index_data.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>This is a bit awkward. Why not combine the two? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>